storage/storage.go

```diff
@@ -128,7 +128,11 @@ func TokenOrg(token string) string {
 
 	// 24 is mongo bson id length
 	if len(token) > 24 {
-		return token[:24]
+		newToken := token[:24]
```

Comment:
Wouldn't it be more clear to have 24 as a constant with the name "MongoBsonIdLength" than having 2x magic numbers + a comment ?

Reply:
done @andrei-tyk 👍